woodwork/column_accessor.py

```diff
@@ -61,6 +61,10 @@ def init(self, logical_type=None, semantic_tags=None,
                                     description=description,
                                     metadata=metadata)
 
+    @property
+    def schema(self):
+        return copy.deepcopy(self._schema)
```

Comment:
I _think_ `copy.deepcopy` does exactly what we need it to--copying for the purposes of the schema property and copying prior to initializing with a schema. So unless I'm missing something here, I'm happy to not implement a deep copy of our own and close the related issue

Reply:
I'd be fine using `copy.deepcopy` for the table as well. That way we can use `_get_subset_schema` for getting actual subsets!

Reply:
Yeah, that seems cleaner!